mnemosyne/core/embeddings.py

```
@@ -95,39 +118,37 @@ def _get_prefix(kind: str) -> str:
-    """True when dense retrieval has been opted out via env var.
-
-    Three flags, in priority order:
-    - MNEMOSYNE_NO_EMBEDDINGS: hard off, used in CI and unit tests that
-      exercise non-embedding code paths
-    - MNEMOSYNE_SKIP_EMBEDDINGS: same intent, shorter alias
-    - MNEMOSYNE_EMBEDDINGS_OFF: same intent, longer alias
-    """
+    """True when dense retrieval has been opted out via env var or config.yaml."""
+    for var in ("MNEMOSYNE_NO_EMBEDDINGS", "MNEMOSYNE_SKIP_EMBEDDINGS", "MNEMOSYNE_EMBEDDINGS_OFF"):
+        if var in os.environ:
+            if bool(os.environ[var]):
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)